### PR TITLE
doc: aries-js-worker release notes

### LINKED ISSUE #1339: Release notes for aries-js-worker

### PR DESCRIPTION
**Pull request by llorllale**
closes #1339 

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The project can also be used as a [DIDComm Router/Mediator](docs/didcomm_router.
 - [Go](docs/go/README.md)
 - [REST](docs/rest/README.md)
   - [Run OpenAPI Demo](docs/rest/openapi_demo.md)
+- [Javascript](cmd/aries-js-worker//README.md)
```
Comment (troyronda) [inline README.md]: extra `/`

File: README.md [hunk 2]
```diff
@@ -22,8 +22,8 @@ API bindings are supplied to enable these environments including:
 
 - Go
 - REST
+- JS / WebAssembly
```
Comment (rolsonquadras) [inline README.md]: Expanding to JavaScript looks better.

File: README.md (hunk 1 above)
Comment (llorllale) [reply README.md]: fixed

File: cmd/aries-js-worker/README.md [hunk 3]
```diff
@@ -11,33 +11,147 @@ To build you will need:
...
+
+```html
+<script src="dist/rest/aries.js"></script>
+```
+
+Then initialize your aries instance:
+
+```js
+const aries = await new Aries.Framework({
+    assetsPath: "/path/serving/the/assets", // still required for assets other than the wasm
```
Comment (rolsonquadras) [inline cmd/aries-js-worker/README.md]: Do we have any doc on these startup params ? May be we can point to https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/agent_cli.md#agent-parameters, if these are similar.

Comment (rolsonquadras) [reply cmd/aries-js-worker/README.md]: ignore - found it https://github.com/hyperledger/aries-framework-go/pull/1373/files#diff-a43de5ddb6dcf66800367a9232d43f90R84.

File: README.md [hunk 4]
```diff
@@ -38,6 +38,7 @@ The project can also be used as a [DIDComm Router/Mediator](docs/didcomm_router.
 - [Go](docs/go/README.md)
 - [REST](docs/rest/README.md)
   - [Run OpenAPI Demo](docs/rest/openapi_demo.md)
+- [Javascript](cmd/aries-js-worker/README.md)
```
Comment (troyronda) [inline README.md]: JavaScript

File: README.md (hunk 2 above)
Comment (llorllale) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (llorllale) [reply README.md]: fixed

**Pull request merged**